<?xml version="1.0"?>
<launch>

    <include file="$(find ros_robot_sim_pkg)/launch/sim_w1.launch" />
    
    <param name="des_pos_x" value= "0.0" />
    <param name="des_pos_y" value= "1.0" />
    
    <node pkg="ros_robot_sim_pkg" type="wall_follow_service.py" name="wall_follower" />
    <node pkg="ros_robot_sim_pkg" type="go_to_point_service.py" name="go_to_point"  />
    <node pkg="ros_robot_sim_pkg" type="bug_as.py" name="bug_action_service" output="screen" />    
   
    <node pkg="ros_robot_sim_pkg" type="acNode_a.py" name="acNode_a" output="screen" />
    <node pkg="ros_robot_sim_pkg" type="srvNode_b.py" name="srvNode_b" output="screen" />
    <node pkg="ros_robot_sim_pkg" type="srvNode_c.py" name="srvNode_c" output="screen" />
    <param name="window_size" type="int" value="100" />
    
</launch>
